### PR TITLE
Visual diff tests: Setup & Status Table tests

### PR DESCRIPTION
**Pull request by margaree**

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ const eleventyNavigationPlugin = require('@11ty/eleventy-navigation');
 const { escapeHtml } = require('markdown-it/lib/common/utils');
 
 module.exports = function(eleventyConfig) {
+	eleventyConfig.addPassthroughCopy('img');
```
Comment (margaree) [inline .eleventy.js:7]: I moved img so that the images could be picked up by the visual-diff tests

File: test/status-table.visual-diff.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+					{
+						'name': 'Not Started with Team',
+						'development': 'Not Started',
+						'owner': 'Great Team'
+					}
+				]
+			};
+
+			window.addEventListener('DOMContentLoaded', () => {
+				const tables = document.querySelectorAll('d2l-component-catalog-status-table');
```
Comment (margaree) [inline test/status-table.visual-diff.html:69]: This section is for setting the components data. I couldn't figure out how to mock the file returned but this worked.

**Review by GeorgePlukov — APPROVED** [review]
LGTM. Do you think we'll try and add some sort of visual diff testing for the live site (not mocked data)? I know there will be edits happening in the README's but might be a nice way to check what is changing on the site before deploys.

**Comment by margaree** [comment]
Yup agreed something like that would be nice. I'm going through the components first and then hoping to look into something bigger picture.

**Pull request merged**